tests/acceptance/helpers/browserConsole.js

```diff
@@ -0,0 +1,32 @@
+import { client } from 'nightwatch-api'
+
+function cleanupLogMessage (message) {
+  return message
+    .replace(/\\u003C/gi, '')
+    .replace(/\\n/g, '\n') // revive newlines
+}
+
+function formatLog (log) {
+  return new Date(log.timestamp).toLocaleTimeString() + ' - ' +
+    log.level + ' - ' +
+    cleanupLogMessage(log.message)
+}
+
+async function getAllLogs () {
+  let logs = []
+  await client.getLog('browser', entries => {
+    logs = entries
+  })
+  return logs
+}
+
+export async function getAllLogsWithDateTime (level = null) {
```

Comment:
```suggestion
export async function getLogsOfLevel (level = null) {
```

Reply:
Here, `level` is just a filter to narrow the results down. If you do not provide any level, it will return unfiltered results. So, I don't prefer the above suggestion.

How about simple `getAllLogs()`? Or, any better suggestions?